src/routes/api/index.js

```diff
@@ -2,11 +2,13 @@ import express from 'express';
 import authenticationRouter from './authentication.routes';
 import requestRouter from './request.routes';
 import profileRouter from './profile.routes';
+import roleRouter from './role.routes';
 
 const apiRouter = express.Router();
 
 apiRouter.use('/auth', authenticationRouter);
 apiRouter.use('/trips', requestRouter);
 apiRouter.use('/profile', profileRouter);
+apiRouter.use('/admin', roleRouter);
```

Comment:
Could you name this handler to `adminRouter` instead of `roleRouter`, coz it fits the endpoint's name `/admin`, or you could just change `/admin` to `/roles`

Reply:
I think the best way is to rename the route to role/assign-role instead of renaming it to admin because it will handle routes related to roles not to admin